src/app/modules/submission/submission.utils.ts

```diff
@@ -758,3 +766,54 @@ export const getAnswersForChild = (
     })
   })
 }
+
+/**
+ * Returns true if the authType is Singpass (with NRIC/FIN) and false if nil or Corppass.
+ */
+export const checkIsIndividualSingpassAuthType = (
+  authType: FormAuthType,
+): boolean => {
+  return (
+    authType === FormAuthType.SP ||
+    authType === FormAuthType.MyInfo ||
+    authType === FormAuthType.SGID ||
+    authType === FormAuthType.SGID_MyInfo
+  )
```

Comment:
Might be terser with this, and it's easier to extend in the future.
```suggestion
  return [
    FormAuthType.SP,
    FormAuthType.MyInfo,
    FormAuthType.SGID,
    FormAuthType.SGID_MyInfo,
  ].includes(authType)
```

Reply:
yeah definitely! will change to this much cleaner code!

Reply:
on second look, seems ive removed this function since corppass is also now supported for single submission. will keep this `includes` function in mind in the future though